schema2Markdown.js

```diff
@@ -0,0 +1,259 @@
+const fs = require('fs-extra');
```

Comment:
If you're going to use `fs-extra`, could you name it `fse`, or something else to differentiate it from the Node `fs` package? I kept reading things like `fs.outputJsonSync` and thinking, "when did fs add *that* method?".